### PR TITLE
Fix injuries command

### PR DESCRIPTION
**Pull request by reddit-tennis**
also greatly simplified its logic.

### CODE REVIEW

**Comment by reddit-tennis** [comment]
image

File: cogs/playerstats.py [hunk 1]
```diff
@@ -105,16 +105,33 @@ async def assists(self, interaction: discord.Interaction, competition: str = 'pl
...
+        }
+        response = requests.get(injuries_url, headers=headers).text
+        parsed_page = BeautifulSoup(response, features="lxml")
+        team_table = parsed_page.find_all('tr', attrs={'class': "player-row team_1"})
+        injured_players = [i.text for i in team_table]
+
+        embed = discord.Embed(color=0x9C824A)
+        embed.set_footer(text=f"Data is sourced from {injuries_url}")
+
+        for player in injured_players:
```
Comment (ash663) [inline cogs/playerstats.py:122]: Can you add a comment indicating what the `player` object looks like? Makes it easier for others to understand the code

Comment (reddit-tennis) [reply cogs/playerstats.py:122]: Will see what I can do as its quite large (that's what she...)
image

**Review by ingenium21 — APPROVED** [review]
feelsrizalman for fixing this before me  
but feelschoudaman for simplifying the logic.

File: cogs/playerstats.py (hunk 1 above)
Comment (ingenium21) [reply cogs/playerstats.py:122]: just do the first player.

Comment (reddit-tennis) [reply cogs/playerstats.py:133]: Added

**Comment by ash663** [comment]
🫡

**Review by ash663 — APPROVED** [review]

**Pull request merged**